addons/web/static/src/xml/base.xml

```diff
@@ -508,7 +508,7 @@
 </t>
 
 <t t-name="Popover" owl="1">
-    <div t-on-click="openPopover" data-popover="1">
+    <div t-att-class="{ 'o_is_open': state.displayed }" t-on-click="_onClick" t-on-o-popover-close="_onPopoverClose">
```

Comment:
prefixed by 'o-'?

Reply:
Those are DOM events that can potentially conflict with any other lib, similar to what we do for CSS classes.

Livechat will need popover, and it can be included on any website.

Reply:
good point